plugins/CoreAdminHome/tests/Integration/Commands/RunScheduledTasksProcessSignalTest.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link    https://matomo.org
+ * @license https://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Plugins\CoreAdminHome\tests\Integration\Commands;
+
+use Piwik\CliMulti\ProcessSymfony;
+use Piwik\Plugins\CoreAdminHome\Tasks;
+use Piwik\Plugins\CoreAdminHome\tests\Fixtures\RunScheduledTasksProcessSignal as RunScheduledTasksProcessSignalFixture;
+use Piwik\Scheduler\Task;
+use Piwik\Tests\Framework\Fixture;
+use Piwik\Tests\Framework\TestCase\IntegrationTestCase;
+
+/**
+ * @group Core
+ * @group CoreAdminHome
+ * @group RunScheduledTasksProcessSignal
+ */
+class RunScheduledTasksProcessSignalTest extends IntegrationTestCase
+{
+    /**
+     * @var RunScheduledTasksProcessSignalFixture
+     */
+    public static $fixture;
+
+    public function setUp(): void
+    {
+        if (!extension_loaded('pcntl') || !function_exists('pcntl_signal')) {
+            $this->markTestSkipped('signal test cannot run without ext-pcntl');
```

Comment:
I don't understand exactly when this might happen, can you elaborate? 

Just worried that for some reason we skip this test without realizing and things break.

Reply:
If you run the tests on a system where the methods might be disabled for example. We have similar checks for other things as well. It might though be useful to start tracking skipped tests on our CI, so we are aware of changes...